spec/models/audit/event/authn/authenticate_spec.rb

```diff
@@ -56,6 +62,12 @@
       expect(subject.severity).to eq(Syslog::LOG_WARNING)
     end
 
+    it 'contains the not-found user field' do
+      expect(subject.structured_data).to match(hash_including({
+          Audit::SDID::AUTH => { authenticator: authenticator_name, service: "rspec:webservice:my-service", user: Audit::Event::NOT_FOUND }
```

Comment:
this line is very long. please change it to:
```
Audit::SDID::AUTH => { 
    authenticator: authenticator_name, 
    service: "rspec:webservice:my-service", 
    user: Audit::Event::NOT_FOUND 
  }
```

verify you don't have any other places where we can't do that.